Signal/src/view controllers/RegistrationViewController.m

```diff
@@ -16,6 +16,8 @@
 
 @interface RegistrationViewController ()
 
+@property (nonatomic) NSString *lastCallingCode;
```

Comment:
nit: At a glance "last" denotes "previous", but I think we mean the currently selected one? Maybe "selectedCallingCode" or just "callingCode"

feel free to ignore, just my initial thought when I saw it being used.